src/renderer/components/+network-services/service-port-component.tsx

```diff
@@ -61,14 +105,32 @@ export class ServicePortComponent extends React.Component<Props> {
   render() {
     const { port } = this.props;
 
+    if (this.forwardPort == null) {
+      this.forwardPort = port.port;
```

Comment:
I think this is generally not a good idea. If the pod or service port is 80 then suggesting the forward be 80 is a problem. Also, this will cause problems for frequently used ports like 8080, when you want to forward more than one service that exposes 8080. 
If `this.forwardPort` is `null` I think the port input field should start as blank (or some placeholder text like "random"  or "arbitrary port"). If the user leaves it blank then Lens does what it did before (lets `kubectl` find a port)

Reply:
Thanks for the suggestion I have added the random port option if a port is not given, with a Random placeholder